src/metaswitch/clearwater/config_manager/test/test_config_type.py

```diff
@@ -124,6 +125,29 @@ def test_get_xml(self):
                                  'path', ]]
         self.assertListEqual(answer.values(), ifcs_expected_script)
 
+@mock.patch('metaswitch.clearwater.config_manager.config_type_class_plugin.os.access')
+@mock.patch('metaswitch.clearwater.config_manager.config_type_class_plugin.subprocess.check_output')
+class TestRphValidation(unittest.TestCase):
```

Comment:
I was about to say that you don't need to have this test here since what this file is really doing is testing the config access plugin, and there is already a test for finding scripts below this one. That said, given that you've overriden the `validate()` function in the RPH plugin it's probably worth having. Once you've addressed the technical debt, you can probably remove this.

Reply:
added comment to make this clear